webview-ui/package.json

```diff
@@ -75,13 +75,12 @@
 		"eslint-plugin-react-hooks": "^4.6.0",
 		"eslint-plugin-storybook": "^0.11.2",
 		"identity-obj-proxy": "^3.0.0",
-		"jest": "^27.5.1",
-		"jest-environment-jsdom": "^27.5.1",
-		"jest-simple-dot-reporter": "^1.0.5",
+		"jest": "^29.7.0",
```

Comment:
Might as well match the jest and ts-jest versions that we use to test the extension.